packages/webdoc-default-template/tmpl/components/member.tmpl

```diff
@@ -1,21 +1,18 @@
 <?js
-    /** Renders a method or function document inline */
```

Comment:
I think we don't have to remove this description.

Reply:
Fixed in 9e7db62055baad1b7af9e1bcbdbfbd3fa02c5520.